google-cloud-kubernetes/config.json

```diff
@@ -0,0 +1 @@
+{"config-changes":{"Xmx":"20G","retrieval.forbackend.concurrency":"5","retrieval.forbackend.loops":"30"},"num-nodes":"1","node-type":"n1-standard-8"}
```

Comment:
This isn't the right way to specify config. 

- No of nodes and type should move to a kubernetes config file
- App config should move to a configmap and referenced while creating a pod